tests/test_bedrock_client.py

```
@@ -0,0 +1,191 @@
+    logs = b"gamerule keepInventory = true\nThere are 1/10 players online:\nVonCrush\n"
+    history = b"Player connected: VonCrush, xuid: 999\n"
+    container = MagicMock()
+    container.attach_socket.return_value._sock = MagicMock()
+    container.logs.side_effect = [logs, history]
+    docker_client = MagicMock()
+    docker_client.containers.get.return_value = container
+    docker_mod = MagicMock()
+    docker_mod.from_env.return_value = docker_client
+
+    with patch.dict("sys.modules", {"docker": docker_mod}):
+        gamerules, players, online, maximum, xuids = client.query_state()
```

Ruff (0.16.1)

[warning] Unpacked variable `maximum` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)